frameworks/engineering/engineering-manager.md

```diff
@@ -0,0 +1,121 @@
+          - "Faz pareamento com Tech Leads no sentido de apoia-los(as) em decisões complexas;"
+          - "Sabe quando é hora de acompanhar de perto quedas de produtividade, engajamento e motivação das pessoas;"
+          - "Usa ferramentas para identificar saúde dos times que trabalha e até da área de tecnologia como um todo;"
+  - name: "agilidade"
+    title: "Agilidade"
+    content:
+      - level: 1
+        criteria:
+          - "Conhece os conceitos de agilidade, do manifesto e seus valores, uma ou mais metodologias ágeis e reconhece a importância de cada uma dessas metodologias;"
+          - "Acompanha cerimônias e práticas dos times pra que consiga identificar e propor as mais adequadas para o contexto;"
+          - "Trabalha em par com o papel de Agile Master ou Agile Coach em ações em conjunto para melhoria contínua dos times;"
+      - level: 2
+        criteria:
+          - "É referência em agilidade para os pares e desenvolvedores(as) do time, trazendo, quando possível, conteúdo e metodologias novas para a empresa;"
+          - "Implementa de forma colaborativa novas formas de trabalho ou práticas do time;"
+          - "Revisa constantemente o que tem funcionado nos times e compartilha com outras pessoas;"
+          - "Tenta fazer desnecessário o máximo que possível o papel de Agile Master ou Agile Coach nos times;"
+  - name: "segurança psicológica"
+    title: "Segurança Psicológica"
+    content:
+      - level: 1
+        criteria:
+          - "Identifica possíveis comportamentos que afastam as pessoas das conversas, dando feedback para pessoas que direta ou indiretamente contribuem pra esse ambiente não seguro nos times;"
+          - "Promove sempre a colaboração e participação de todas as pessoas em reuniões, cerimônias ou interações, dando chance pra que elas falem e evitando monopolização do discurso;"
+          - "Identifica quaisquer comportamentos que fortaleçam uma cultura de culpa por erros, trazendo sempre uma visão de aprendizado com as falhas;"
+      - level: 2
+        criteria:
+          - "Sabe como atuar em cada uma das circunstâncias de insegurança nos times;"
+          - "Acompanha, com certa frequência, o que está impedindo o time de ser aberto e honesto consigo mesmo;"
+          - "Promove uma cultura de reconhecimento das entregas, através de ferramentas ou mesmo na interação das pessoas;"
+          - "Aponta possíveis problemas culturais entre os times, no que se refere ao ambiente seguro para o aprendizado, e traz possíveis soluções para os mesmos;"
+  - name: "estratégia e negócios"
+    title: "Estratégia e Negócios"
+    content:
+      - level: 1
+        criteria:
+          - "Está alinhado(a) com stakeholders sobre possíveis futuras iniciativas de negócio e como a área de tecnologia pode ser impactada nisso;"
+          - "Ajuda a definir novas estruturas de times, enxergando necessidade de crescimento;"
```

Comment:
Além disso, acho que rola de explicitar que um dos papéis é propor/promover reflexão sobre quais pessoas seria mais interessante fazer parte de quais times, promover essa questão do "pessoas certas nos lugares certos". Algo desse tipo:
`- Propõe a alocação das pessoas nas estruturas dos times, procurando identificar quem são as pessoas mais adequadas  para atacar os problemas/objetivos daquele time.`

Que tal?

Reply:
Podemos adicionar a esse texto algo mais específico. Eu tinha imaginado que "Ajuda a definir novas estruturas de times" contemplaria exatamente esse cenário que você explicou. Mas acho que vale acrescentar e deixar mais claro isso sim!